tls/s2n_ktls.c

```diff
@@ -151,7 +259,11 @@ int s2n_connection_ktls_enable_recv(struct s2n_connection *conn)
     }
 
     POSIX_GUARD_RESULT(s2n_ktls_configure_socket(conn, S2N_KTLS_MODE_RECV));
+    POSIX_GUARD_RESULT(s2n_ktls_configure_crypto(conn, S2N_KTLS_MODE_RECV));
+
     conn->ktls_recv_enabled = true;
+    /* kTLS now handles I/O for the connection */
+    conn->recv = NULL;
```

Comment:
This could lead to a seg fault. Better to point it to a method that returns an error.